src/edges_analysis/analysis/filters.py

```diff
@@ -1338,14 +1339,17 @@ def day_rms_filter(
             gha=gha[mask],
             bins=np.array([gha_min, gha_max]),
         )
+        freq_mask = (data.raw_frequencies >= f_low) & (data.raw_frequencies <= f_high)
         if weighted:
             rms = np.sqrt(
-                averaging.weighted_mean(data=mean_r[0] ** 2, weights=mean_w[0])[0]
+                averaging.weighted_mean(
+                    data=mean_r[0, freq_mask] ** 2, weights=mean_w[0, freq_mask]
+                )[0]
             )
         else:
-            rms = np.sqrt(np.mean(mean_r[0] ** 2))
-
+            rms = np.sqrt(np.mean(mean_r[0, freq_mask] ** 2))
+        print(rms)
```

Comment:
let's take out the print statement (you can put in a logger statement if you require)